src/views/Home/utils/utils.tsx

```diff
@@ -19,7 +19,9 @@ const tooltipLabels: { [key: string]: string } = {
   psm: 'PSM',
   liquidationIncome: 'Liquidation Income',
   fees: 'Fees',
-  daiSpent: 'DAI Spent',
+  dsr: 'DSR Cost',
+  // eslint-disable-next-line spellcheck/spell-checker
+  daiSpent: 'USDS/DAI Expensed',
```

Comment:
"Would it be possible to add this term to the dictionary file and remove the // eslint-disable-next-line spellcheck/spell-checker comment? This way, we can keep the codebase cleaner.

Reply:
Done